pmlc/rt/runtime.h

```diff
@@ -46,13 +47,32 @@ class Device {
 // evaluating a PlaidML program.  These are typically single-instanced within a
 // process, constructed once as needed (typically well after process
 // initialization) and then held onto by the global runtime registry.
+//
+// N.B. The Runtime lifecycle is a little unusual.
+//
+//      Typically, Runtime will be instantiated (via RuntimeRegistration) at
+//      static initialization time, i.e. before main() is invoked.  Since this
+//      is an unusual process environment in which various global objects may or
+//      may not be initialized, Runtime implementations may defer complicated
+//      initialiation to their init() method; the contract with the caller is
```

Comment:
I don't think we should say they 'may' , but they 'must'. All initialization should happen in `init()`. Could we drop the 'unusual' business?

Reply:
Done